Git.hub/Client.cs

```diff
@@ -174,6 +174,27 @@ public List<Repository> searchRepositories(string query)
             return repos.Repositories.Select(r => r.ToV3(_client)).ToList();
         }
 
+        /// <summary>
+        /// Get the issues assigned to the authenticated user.
+        /// <see href="https://docs.github.com/fr/rest/issues/issues?apiVersion=2022-11-28#list-issues-assigned-to-the-authenticated-user">GitHub issue api</see>
+        /// </summary>
+        /// <param name="state">state of the issues returned. Choice: "open", "closed", "all"</param>
+        /// <param name="includePullRequests">true to include pull requests; otherwise, false.</param>
+        /// <returns>a list of issues assigned to the authenticated user.</returns>
+        public List<Issue> GetAssignedIssues(string state = "open", bool includePullRequests = false)
```

Comment:
I'd caution against returning mutable collections. Consider returning a read-only list.

Reply:
I have added a refactoring commit to stay consistent and do the same for all the lists returned.